pkg/utils/common/common.go

```diff
@@ -21,7 +21,7 @@ const (
 	DEFAULT_RESYNC_PERIOD  = 30 * time.Second
 	// comments push this over the edge a little when dealing with white space
 	// as en env var it can be disabled by setting to "" or can be improved!
-	JaasConfigSyntaxMatchRegExDefault = `^(?:(\s*|(?://.*)|(?s:/\*.*\*/))*\S+\s*{(?:(\s*|(?://.*)|(?s:/\*.*\*/))*\S+\s+(?i:required|optional|sufficient|requisite)+(?:\s*\S+=\S+\s*)*\s*;)+(\s*|(?://.*)|(?s:/\*.*\*/))*}\s*;)+\s*\z`
+	JaasConfigSyntaxMatchRegExDefault = `^(?:(\s*|(?://.*)|(?s:/\*.*\*/))*\S+\s*{(?:(\s*|(?://.*)|(?s:/\*.*\*/))*\S+\s+(?i:required|optional|sufficient|requisite)+(?:\s*\S+\s*=((\s*\S+\s*)|("[^"]*")))*\s*;)+(\s*|(?://.*)|(?s:/\*.*\*/))*}\s*;)+\s*\z`
```

Comment:
This regex: (?i:required|optional|sufficient|requisite)+
the '+' suggest one or more such words, should it be just once? with that the following could be legal
'a_realm { a_good_login_module optionalsufficient noOp=true; };'
?

Reply:
good catch, that + needs to be removed. Will add a test and proceed. thanks!

Reply:
sorted